src/components/ProtVista/Popup/Residue/index.js

```diff
@@ -29,25 +28,20 @@ const ProtVistaResiduePopup = ({ detail, sourceDatabase } /*: Props */) => {
   return (
     <section>
       <h6>
+        Residue in {sourceDatabase} -{' '}
         {accession.startsWith('residue:')
           ? accession.split('residue:')[1]
           : accession}
-        {description && <p>[{description}]</p>}
       </h6>
+      {description && <p>[{description}]</p>}
 
-      <div>
-        <div>
-          Residue in {sourceDatabase}{' '}
-          {type && type !== 'residue' && type.replace('_', ' ')}
-        </div>
-        <ul>
-          <li>
-            Position: {start}
-            {end && end !== start ? <>-{end}</> : null}
-          </li>
-          <li>Residue: {residue}</li>
-        </ul>
-      </div>
+      <ul>
+        <li>
+          Position: {start}
+          {end && end !== start ? <>-{end}</> : null}
+        </li>
+        <li>Residue: {residue}</li>
```

Comment:
Is `end !== start` possible? The only case where I believe it could happen is if there are several residue annotations in a row, but in this case, would `residue` be a string of all the residues annotated?

Also, maybe the position and the residue code could be on the same line (not in two different list items)?